lib/translation_maps/locations.tsv

```diff
@@ -449,4 +449,4 @@ SSRC-FIC-S	GREEN	GRE-DOCS-MICROFICHE
 SDR	LAW	LAW-SDR
 SDR	BUSINESS	BUS-SDR
 SDR	HOOVER	HILA-SDR
-SDR	SUL	SUL-SDR
+INTERNET	SUL	SUL-SDR
```

Comment:
(Copied from the outdated thread)

I thought we were trying to keep our locations.tsv in sync with what libsys is using.

Reply:
🤷 are we documenting that somewhere?

Reply:
Sure, anywhere you think would be appropriate. It seems like a very good idea not to change that upstream data.